delivery.yaml

```diff
@@ -8,6 +8,8 @@ pipeline:
   commands:
   - desc: build-push
     cmd: |
+      apt-get update
+      apt-get install -y jq
```

Comment:
It is not actually required for the build in the pipeline:
The `$$HOME/.docker/config.json` does not exist in the pipline image so the content is echoed directly without using jq
https://github.com/zalando/skipper/blob/1a1d1eb186e495b7e67f9b10d278b11652d44f89/packaging/Makefile#L61